src/modules/app/components/Navigator.js

```diff
@@ -74,12 +78,85 @@ export const AppStack = createStackNavigator(
   }
 )
 
-export const LandingStack = createSwitchNavigator(
-  {
-    'Landing': LandingContainer,
-    'App': AppStack
-  }
-)
+export const LandingStack = createSwitchNavigator({
+  // TODO replace this ActivityIndicator, NATIVE-211
+  'Loading': ActivityIndicator,
+  'Landing': LandingContainer,
+  'App': AppStack
+})
 
 const AppContainer: NavigationContainer<NavigationState, {}, {}> = createAppContainer(LandingStack)
-export default AppContainer
+
+type PropsType = {|
+  fetchCategory: (cityCode: string, language: string, key: string) => void,
+  clearCategory: (key: string) => void,
+  fetchCities: () => void
+|}
+
+class Navigator extends React.Component<PropsType> {
+  localizationSettings: LocalizationSettings
+  navigator: ?NavigationContainer<NavigationState, {}, {}>
+
+  constructor (props: PropsType) {
+    super(props)
+    this.localizationSettings = new LocalizationSettings()
+  }
+
+  componentWillMount () {
+    this.props.fetchCities()
```

Comment:
Why do we need to fetchCities here?

Reply:
Because the fetch is otherwise never called, this was done in the Landing page before but since we skip the Landing page if there is a selected city, the fetch will not happen. We could of course also do this in `loadCityContent`

Reply:
But then we would have to call it at two places.